### ISSUE TITLE
Kaggle JSON (in different place)

### ISSUE THREAD

**Issue by duchaba** [JovianML/opendatasets#8]
[aakashns](https://github.com/JovianML/opendatasets/commits?author=aakashns)

This is a useful repository. Thank you for sharing.

FYI: Digging into the code, the method "read_kaggle_creds()" looks for the kaggle.json at the current directory, "./kaggle.json", but most of the time, the kaggle.json is stored at "~/.kaggle/kaggle.json"

Thanks.

**Comment by aakashns** [comment]
Makes sense, we should also look for `kaggle.json` in `~/.kaggle/kaggle.json`. We'll pick up this enhancement soon.

**Comment by youyinnn** [comment]
updates?

**Comment by DmytroLitvinov** [comment]
Just following course of learning ML and that library was mention to download dataset 👍  Thanks for library.

Would be really nice to have that feature :)
